azure-devops/azext_devops/dev/team/user.py

```diff
@@ -0,0 +1,108 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azext_devops.vstsCompressed.member_entitlement_management.v4_1.models.models import (AccessLevel,
+                                                                                          GraphUser,
+                                                                                          JsonPatchOperation)
+from azext_devops.dev.common.services import (get_member_entitlement_management_client,
+                                              resolve_instance)
+from azext_devops.dev.common.identities import resolve_identity_as_id
+
+
+def get_user_entitlements(top=None, skip=None, organization=None, detect=None):
+    """List users for an organization.
+    :param int top: Maximum number of the users to return. Max value is 10000. Default value is 100
```

Comment:
If we put `top=100` instead of `top=None` then knack will take care of this in the standard format
maybe we can use that